static/js/echo.js

```diff
@@ -109,6 +114,10 @@ exports.try_deliver_locally = function try_deliver_locally(message_request) {
         return;
     }
 
+    if (exports.is_slash_command(message_request.content)) {
+        return;
+    }
```

Comment:
This is in the wrong place; we should do it before calling `local_message.get_next_id`.

Reply:
(fixed)